### DISCUSSION TITLE
Support Authelia OIDC

### DISCUSSION THREAD

**Discussion by dlford** [nextauthjs/next-auth#10409] [Ideas]
### Goals

1. Add support [Authelia](https://www.authelia.com/configuration/identity-providers/openid-connect/clients/) as an authentication provider

### Background

Currently Authelia *may* work as a generic OIDC provider, there is one stale issue that was never addressed (#7990).

Ultimately, I want Authelia supported explicitly as I want to use it with LinkWarden, which is utilizing Next-Auth but not providing the option for a generic provider.

### Proposal

I am willing to contribute to this feature.

**Comment by LarssonOliver** [comment]
I'd also be very happy to see explicit support for Authelia!

**Comment by dlford** [comment]
Just an update; I've been quite busy lately, if anybody wants to pick this up they are welcome to, if not I'll eventually get some time to take a crack at it.
